internal/config/env.go

```diff
@@ -1,11 +1,20 @@
 package config
 
 import (
+	"log"
 	"os"
 )
 
 type Environment map[string]string
 
+func (e Environment) SetIfUndefined(k string, v string) {
```

Comment:
```suggestion
// Set will only set the given environment key and value if it isn't already set
// within the Environment.
func (e *Environment) Set(k string, v string) {
```
This will not do anything or set anything if it is not a pointer.

Reply:
Applied and fixed